tools/ci_build/github/linux/run_build.sh

```diff
@@ -53,7 +53,16 @@ else
             --use_cuda \
             --cuda_home /usr/local/cuda \
             --cudnn_home /usr/local/cudnn-$_CUDNN_VERSION/cuda $BUILD_EXTR_PAR
-    elif [ $BUILD_DEVICE = "tensorrt" ]; then
+    elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
+        if [ $BUILD_DEVICE = "tensorrt-v7.1" ]; then
+            CUR_PWD=$(pwd)
+            cd $SCRIPT_DIR/../../../../
+            chmod 775 .gitmodules
+            git config --file=.gitmodules submodule.cmake/external/onnx-tensorrt.branch 7.1
+            git submodule update --init --remote cmake/external/onnx-tensorrt
+            COMMON_BUILD_ARGS=${COMMON_BUILD_ARGS/"--skip_submodule_sync"}
```

Comment:
In our CI build, we should always use "--skip_submodule_sync"

Reply:
if I didn't remove "--skip_submodule_sync", CI will fail with following error message:

CMake Error at CMakeLists.txt:627 (add_subdirectory):
  The source directory

    /onnxruntime_src/cmake/external/googletest

  does not contain a CMakeLists.txt file.

Reply:
![image](https://user-images.githubusercontent.com/856316/122471801-8df91780-cf74-11eb-905b-7fffca35ac6f.png)